tools/topology/sof-tgl-sdw-max98373-rt5682.m4

```diff
@@ -56,9 +56,13 @@ define(`SMART_BE_ID', 2)
 # Playback related
 define(`SMART_PB_PPL_ID', 3)
 define(`SMART_PB_CH_NUM', 2)
-define(`SMART_TX_CHANNELS', 4)
-define(`SMART_RX_CHANNELS', 8)
-define(`SMART_FB_CHANNELS', 8)
+# channel number for playback on sdw link
+define(`SMART_TX_CHANNELS', 2)
+# channel number for I/V feedback on sdw link
+define(`SMART_RX_CHANNELS', 4)
+# channel number of feedback stream from
```

Comment:
do you mean echo reference?

Reply:
I agree with you. I found feedback in comments for FB.  call for @keyonjie for help

Reply:
I think it should be Feedback for what is now RX, and EchoRef for what is now FB.

Reply:
@plbossart @plbossart comments as below

# DAI & smart_amp algorithm related
# channel number for playback on sdw link
define(`SMART_TX_CHANNELS', 2)
# channel number for capture on sdw link
define(`SMART_RX_CHANNELS', 4)

# Smart_amp algorithm specific
# channel number for feedback provided to smart_amp algorithm
define(`SMART_FB_CHANNELS', 4)

# Playback PCM stream related
define(`SMART_PB_CH_NUM', 2)

# Capture (EchoRef) PCM stream related
define(`SMART_REF_CH_NUM', 4)